controllers/post-comment.controller.js

```diff
@@ -0,0 +1,17 @@
+const { insertComment } = require("../models/insert-comment.model");
+
+exports.postComment = (request, response, next) => {
+  const { article_id } = request.params;
+  const newComment = request.body;
+
+  insertComment(article_id, newComment)
+    .then((comment) => {
+      if (newComment.body.length === 0) {
+        response.status(400).send({ message: "Cannot post an empty comment!" });
+      }
```

Comment:
Error handling should not be ideally done in the controller. This should be handled in the model and rejected so it can go into the .catch and handled by your middleware handlers